invenio_records_rest/schemas/fields/generated.py

```diff
@@ -42,6 +45,28 @@ def __init__(self, *args, **kwargs):
         # Setting "missing" to some value forces the call to ``.deserialize``
         kwargs['missing'] = GeneratedValue
         super(ForcedFieldDeserializeMixin, self).__init__(*args, **kwargs)
+    if marshmallow_version[0] >= 3:
```

Comment:
Add a comment here as well explaining why we need this kind of behaviour of `deserialize` and where does it come from